packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -2725,33 +2753,85 @@ export class ContainerRuntime
 			throw e;
 		}
 	}
+
+	/**
+	 * Processes messages that are not intended for the runtime layer.
+	 * The function today (July 2024) is identical to the 'processRuntimeMessage' function, but will progressively be updated to
```

Comment:
Well it looks like you removed `validateAndProcessRuntimeMessage` already, right?